File: src/main/java/com/ject/studytrip/trip/infra/jpa/TripReportStudyLogJpaRepository.java
package com.ject.studytrip.trip.infra.jpa;

import com.ject.studytrip.trip.domain.model.TripReportStudyLog;
import org.springframework.data.jpa.repository.JpaRepository;

public interface TripReportStudyLogJpaRepository extends JpaRepository<TripReportStudyLog, Long> {}

File: src/main/kotlin/com/ject/studytrip/trip/infra/jpa/TripReportStudyLogJpaRepository.kt
package com.ject.studytrip.trip.infra.jpa

import com.ject.studytrip.trip.domain.model.TripReportStudyLog
import org.springframework.data.jpa.repository.JpaRepository

interface TripReportStudyLogJpaRepository : JpaRepository<TripReportStudyLog, Long>
